charms/openfga-k8s/src/charm.py

```diff
@@ -140,42 +150,26 @@ def __init__(self, *args):
         )
 
         # Ingress relation
-        self.ingress = IngressPerAppRequirer(
-            self, relation_name="ingress", port=8080
-        )
+        self.ingress = IngressPerAppRequirer(self, relation_name="ingress", port=8080)
         self.framework.observe(self.ingress.on.ready, self._on_ingress_ready)
-        self.framework.observe(
-            self.ingress.on.revoked, self._on_ingress_revoked
-        )
+        self.framework.observe(self.ingress.on.revoked, self._on_ingress_revoked)
 
         # Database relation
         self.database = DatabaseRequires(
             self,
             relation_name="database",
             database_name="openfga",
         )
-        self.framework.observe(
-            self.database.on.database_created, self._on_database_event
-        )
+        self.framework.observe(self.database.on.database_created, self._on_database_event)
         self.framework.observe(
             self.database.on.endpoints_changed,
             self._on_database_event,
         )
-        self.framework.observe(
-            self.on.database_relation_broken, self._on_database_relation_broken
-        )
-
-        port_http = ServicePort(
-            8080, name=f"{self.app.name}-http", protocol="TCP"
-        )
-        port_grpc = ServicePort(
-            8081, name=f"{self.app.name}-grpc", protocol="TCP"
-        )
-        self.service_patcher = KubernetesServicePatch(
-            self, [port_http, port_grpc]
-        )
+        self.framework.observe(self.on.database_relation_broken, self._on_database_relation_broken)
 
-        self.state = PeerRelationState(self.model, self.app, "openfga-peer")
+        port_http = ServicePort(8080, name=f"{self.app.name}-http", protocol="TCP")
+        port_grpc = ServicePort(8081, name=f"{self.app.name}-grpc", protocol="TCP")
+        self.service_patcher = KubernetesServicePatch(self, [port_http, port_grpc])
```

Comment:
I think Ali might have a way of handling multiple ingress ports without the K8ServicePatch now. Something for the future.